it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/ruleeditor/basic/.content.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:deviceGroups="[/etc/mobile/groups/responsive]"
+        cq:lastModified="{Date}2024-03-01T11:49:33.311Z"
+        cq:lastModifiedBy="admin"
+        cq:template="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"
+        jcr:language="en"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="basic"
+        sling:configRef="/conf/forms/core-components-it/samples/ruleeditor/basic/"
+        sling:resourceType="forms-components-examples/components/page">
+        <guideContainer
+            fd:version="2.1"
+            jcr:lastModified="{Date}2024-03-01T11:40:51.518Z"
+            jcr:lastModifiedBy="admin"
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="forms-components-examples/components/form/container"
+            clientLibRef="corecomponent.it.ruleeditor,corecomponent.it.customfunction"
```

Comment:
What's the rationale behind having two client libraries? If they serve different purposes, shouldn't we name them accordingly to reflect their use cases? Currently, `corecomponent.it.ruleeditor` and `corecomponent.it.ruleeditor` aren't very intuitive in terms of understanding their intended functions.

Reply:
It is to test the functionality that form can handle more than one clientlib category and we get a merged list of custom functions on form.

Reply:
In that case, can we rename the client library category name ? They are not readable.

Reply:
Done.
I am keeping category name as corecomponent.it.customfunction and corecomponent.it.customfunction2
also added a test case to make it clear.
`it("should have merged custom function list registered in FunctionRuntime from both clientlibs", () => {
            expect(formContainer, "formcontainer is initialized").to.not.be.null;
            let func;
            cy.window().then(win => {
                func = win.FormView.FunctionRuntime.customFunctions.testFunction1; // from corecomponent.it.customfunction
                expect(func).to.not.be.null;
                expect(func).to.not.be.undefined;

                func = win.FormView.FunctionRuntime.customFunctions.testSubmitFormPreprocessor; // from corecomponent.it.customfunction
                expect(func).to.not.be.null;
                expect(func).to.not.be.undefined;

                func = win.FormView.FunctionRuntime.customFunctions.testSetProperty; // from corecomponent.it.customfunction2
                expect(func).to.not.be.null;
                expect(func).to.not.be.undefined;
            })
        })`